src/PIL/ImageFile.py

```diff
@@ -139,6 +139,9 @@ def get_format_mimetype(self):
         if self.format is not None:
             return Image.MIME.get(self.format.upper())
 
+    def _use_im_values(self):
+        return self.tile is None and self.im is not None
```

Comment:
What is purpose of `self.tile` check?

Reply:
@radarhere proposed that change for `ImageFile` in a [pull request](https://github.com/Yay295/Pillow/pull/9) they made for the previous code. The issue is that when seeking to another frame, the `Image` values may not match the `Image.im` values. Using `not self.tile` like in their pull request wasn't working for me though, so I changed it to `self.tile is None`, which after looking into it more, probably isn't correct.